src/autocomplete/EmojiProvider.tsx

```diff
@@ -33,9 +34,13 @@ import { TimelineRenderingType } from '../contexts/RoomContext';
 
 const LIMIT = 20;
 
+// The delimiter used to start and end emoji shortcodes.
+const EMOJI_DELIMITER = ':';
```

Comment:
not sure this is safe/best to break out to a constant if we're using it in both regex and strings, as if we were to try and use the value of breaking it out to a variable then we might need to escape the new character in regex but not in strings. 

I'm generally confident that this doesn't need to be a variable though, given the chances of it changing are incredibly slim. It would require a global effort to represent emoji differently for us to change it.

Reply:
Removed.